Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -541,12 +564,21 @@ extension AVFoundationPlayback {
         return self.options[kMinDvrSize] as? Double ?? 60.0
     }
 
-    open override var usingDVR: Bool {
+    open override var isDvrInUse: Bool {
         guard playbackType == .live else { return false }
         guard let currentTime = player?.currentTime().seconds else { return false }
         return seekableTimeRanges.first(where: { $0.timeRangeValue.end.seconds > currentTime}) != nil
     }
 
+    open override var isDvrAvailable: Bool {
+        guard playbackType == .live else { return false }
+        return seekableTimeRanges.first(where: { $0.timeRangeValue.duration.seconds >= minDvrSize}) != nil
```

Comment:
Why are we considering a single seekableTimeRange? The total duration should be taken into account, not a single range.